doc/source/how-to/releasing.rst

```diff
@@ -212,39 +212,133 @@ script. If you are operating out of a GitHub CI pipeline, email the PyAnsys Core
 team at `pyansys.core@ansys.com <mailto:pyansys.core@ansys.com>`_ for the
 required ``PAT`` user name and ``PYANSYS_PYPI_PRIVATE_PAT`` password.
 
-Assuming that you are already in a Python repository's ``dist/`` directory that
-contains your wheel, source distribution, or both, you can upload to the private
-repository with:
+For uploading packages, `Twine <https://pypi.org/project/twine/>`_ is required.
+Install this tool by running:
 
-.. code-block:: bash
+.. code-block:: text
```

Comment:
Why text instead of bash? It should be a bash command right?

Reply:
I usually apply `text` syntax since the color highlighting for `bash` output is a bit weird in my opinion. I think it is also possible to use `console`.

Available lexers are collected in [this website](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Codeblocks.html#available-lexers).

I think we could use `shell`.

Reply:
``shell`` is fine! :) However, we should try and adapt everywhere if that's the case. At least for the files affected by this PR. Can you handle that @jorgepiloto?